_docs/1.0.0/usage-schema.md

```diff
@@ -0,0 +1,618 @@
+
+For cases when the number is formatted in some other manner, patterns can come for rescue. The parsing is executed using
+the *Java* class `DecimalFormat`, whose [documentation](#https://docs.oracle.com/javase/7/docs/api/java/text/DecimalFormat.html)
+provides the most comprehensive explanation of patterns and their usage.
+
+Pattern contains a positive and negative subpattern, for example, `#,##0.00;(#,##0.00)`. Each subpattern has a prefix,
+numeric part, and suffix. The negative subpattern is optional; if absent, then the positive subpattern prefixed with the
+minus sign ('-' in most locales) is used as the negative subpattern. That is, `0.00` alone is equivalent to `0.00;-0.00`.
+If there is an explicit negative subpattern, it serves only to specify the negative prefix and suffix; the number of
+digits, minimal digits, and other characteristics are all the same as the positive pattern. That means that `#,##0.0#;(#)`
+produces precisely the same behavior as `#,##0.0#;(#,##0.0#)`.
+
+The prefixes, suffixes, and various symbols used for infinity, digits, thousands separators, decimal separators, etc. may
+be set to arbitrary values. However, care must be taken that the symbols and strings do not conflict, or parsing will be
+unreliable. For example, either the positive and negative prefixes or the suffixes must be distinct for parsing to be able
+to distinguish positive from negative values. Another example is that the decimal separator and thousands separator should
+be distinct characters, or parsing will be impossible.
+
+| Symbol | Location              | Meaning                                                                                                                                                             | Metadata to change                                                                 |
+|--------|-----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|
+| `0`    | Number                | Digit                                                                                                                                                               |                                                                                    |
+| `#`    | Number                | Digit, zero shows as absent                                                                                                                                         |                                                                                    |
+| `.`    | Number                | Decimal separator or monetary decimal separator                                                                                                                     | [Decimal separator](#deciaml_separator) NB!<sup>[*](#pattern-parsing-star)</sup>   |
```

Comment:
What's a `monetary decimal separator`? How is it different from a regular `Decimal separator`?